Mobile-Development/Contribute/Readme.md

```diff
@@ -17,6 +17,7 @@
 - [mediapipe](https://github.com/google/mediapipe)
 - [mirai](https://github.com/mamoe/mirai)
 - [MMKV](https://github.com/Tencent/MMKV)
+- [NewPipe](https://github.com/TeamNewPipe/New Pipe)
```

Comment:
This link is not working

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Brittle